linux/sample_driver.sh

```diff
@@ -9,11 +9,11 @@ DIRECTORY="/tmp/csvs"
 
 echo "Directory: $DIRECTORY."
 
-# Loop through every CSV file in the directory and call sendcsv.bat
+# Loop through every CSV file in the directory and call send_csv.sh
 DIRECTORY="$DIRECTORY/*.csv"
 for f in $DIRECTORY; do
 	echo "Uploading: $f."
-	./sendcsv.sh "$f"
+	./send_csv.sh "$f"
```

Comment:
ah, i think i understand that generate_script.rb creates a send_csv.sh file. How do they run generate_script? should you delete the sendcsv.sh file thats currently in the repo?